README.md

```diff
@@ -208,7 +208,7 @@ status, err := task.Wait(context)
 
 ### Checkpoint and Restore
 
-If you have [criu](https://criu.org/Main_Page) installed on your machine you can checkpoint and restore containers and their tasks.  This allows you to clone and/or live migrate containers to other machines.
+If you have [criu](https://criu.org/Main_Page) installed on your machine you can check and restore containers and their tasks.  This allows you to clone and/or live migrate containers to other machines.
```

Comment:
"checkpoint" is the correct word here

Reply:
@samuelkarp Thanks for tips, the `checkpoint` is a noun,  I understand it needs to be checked

Reply:
"checkpoint" is being used as a verb here.  A more verbose way to write this and use "checkpoint" as a noun would be "you can create a checkpoint".